keyboards/dtd/ldmlKeyboard.dtd

```diff
@@ -84,6 +85,8 @@ Except as contained in this notice, the name of a copyright holder shall not be
 
 <!ELEMENT display EMPTY >
 <!ATTLIST display to CDATA #REQUIRED >
+    <!--@DEPRECATED-->
+<!ATTLIST display mapOutput CDATA #REQUIRED >
```

Comment:
This would not be backward compatible

Reply:
Currently no layouts in the repository use the <display> element.

Reply:
there could, I suppose, be layouts not in the repository which use `<display>`, it could be noted as a migration issue.

Reply:
do we need to add `<!--@VALUE-->` to `to` to prevent it from being distinguishing?
@macchiati ideas?

Reply:
Yes.